repl/src/main/scala/com/cloudera/livy/repl/ProcessInterpreter.scala

```diff
@@ -55,6 +55,7 @@ abstract class ProcessInterpreter(process: Process)
     // jobs against pyspark or sparkr, so just return null here.
     null
   }
+
```

Comment:
`getOrCreate` should return SparkContext which is created in the pyspark/spark initialization script.